.deployment.config.json

```diff
@@ -1,67 +1,74 @@
 {
-    "product": "ui-kit",
-    "team_name": "searchui",
-    "general": {
-        "aws_regions": {
-            "sequential": [
-                "us-east-1"
-            ]
-        },
-        "environments_order": {
-            "sequential": [
-                "dev"
-            ]
-        },
-        "team_jenkins": "searchuibuilds",
-        "start_environment_automatically": false,
-        "notifications": {
-            "slack_channels": [
-                "#searchuibuilds"
-            ]
-        }
-    },
-    "ordered_phases": [
-        {
-            "id": "deploy-headless",
-            "s3": {
-                "bucket": "{terraform.infra.infra.bucket_binaries}",
-                "directory": "proda/StaticCDN/headless/latest",
-                "source": "packages/headless/dist/browser",
-                "parameters": {
-                    "acl": "public-read"
-                }
-            }
-        }
-    ],
-    "certifiers": {
-        "dev": [
-            {
-                "system_certifier": "snyk"
-            },
-            {
-                "system_certifier": "veracode"
-            }
-        ]
-    },
-    "snyk": {
-        "org": "coveo-jsui",
-        "no_container_images": true,
-        "configurations": [
-            {
-                "directory": "packages/headless",
-                "project_name": "headless-engine"
-            }
-        ]
+  "product": "ui-kit",
+  "team_name": "searchui",
+  "general": {
+    "aws_regions": {
+      "sequential": ["us-east-1"]
     },
-    "veracode": {
-        "sandbox_name": "UI Kit (JS UI)",
-        "scan_include_patterns": "*"
+    "environments_order": {
+      "sequential": ["dev", "qa", "prd"]
     },
-    "observatory": {
-        "no_endpoint": true
-    },
-    "package_rollout": {
-        "only_consider_changesets_after": "b244fe702d8e96d016a52715e92c8131acfde3ba"
+    "team_jenkins": "searchuibuilds",
+    "start_environment_automatically": false,
+    "notifications": {
+      "slack_channels": ["#searchuibuilds"]
+    }
+  },
+  "ordered_phases": [
+    {
+      "id": "deploy-headless-to-s3-latest",
+      "s3": {
+        "bucket": "{terraform.infra.infra.bucket_binaries}",
+        "directory": "proda/StaticCDN/headless/latest",
+        "source": "packages/headless/dist/browser",
+        "parameters": {
+          "acl": "public-read"
+        }
+      }
     },
-    "deployment_config_version": 1
-}
\ No newline at end of file
+    {
+        "id": "deploy-headless-to-s3-version",
+        "s3": {
+          "bucket": "{terraform.infra.infra.bucket_binaries}",
+          "directory": "proda/StaticCDN/headless/v$[PRERELEASE]",
+          "source": "packages/headless/dist/browser",
+          "parameters": {
+            "acl": "public-read"
+          },
+          "prd": {
+            "directory": "proda/StaticCDN/headless/v$[VERSION]"
+          }
+        }
+      },
+    {
+      "id": "production-release",
```

Comment:
The production release step of the pipeline.

It is `disabled:true` in dev and qa, and `disabled:false` in prod